doc/getting-started/tutorial.md

```diff
@@ -0,0 +1,100 @@
+#Tutorial
```

Comment:
Made the adjustments. Refer to the [new commit](https://github.com/gradle/kotlin-dsl/pull/382/commits/d09fee34d7ffe6160414d2b9f9e0434e99a7b3b0).